rust/datafusion/src/physical_plan/mod.rs

```diff
@@ -126,10 +126,10 @@ impl ColumnarValue {
         }
     }
 
-    fn into_array(self, batch: &RecordBatch) -> ArrayRef {
+    fn into_array(self, num_rows: usize) -> ArrayRef {
```

Comment:
good. I had the same concern when reviewing this the other day.

Reply:
yeah this is a nice change